Components/Endstop.py

```
@@ -1,6 +1,6 @@
 import RPi.GPIO as GPIO          # Import GPIO library for Raspberry Pi
 from time import sleep, time     # Import sleep for delays and time for timestamps
-import constans
+import Components.constans
```

Ruff (0.11.9)

`Components.constans` imported but unused

Remove unused import: `Components.constans`

(F401)